addons/website_profile/views/website_profile.xml

```diff
@@ -377,7 +377,7 @@
             </t>
         </div>
         <div class="mb-3" t-if="not user.badge_ids">
-            <p class="text-muted">No badge yet</p>
+            <p class="text-muted">No badges yet!</p>
         </div>
         <div class="text-right">
```

Comment:
Don't you need the same trick here?

`#{'d-inline-block pull-right' if not not user.badge_ids and badge_category else ''}`